config/characterizeImage.py

```diff
@@ -7,24 +7,9 @@
 
 obsConfigDir = os.path.dirname(__file__)
 
-# Cosmic rays
-# These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.repair.cosmicray.nCrPixelMax = 100000
-config.repair.cosmicray.cond3_fac2 = 0.4
-
 # PSF determination
 # These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
 config.measurePsf.reserve.fraction = 0.2
-config.measurePsf.starSelector["objectSize"].sourceFluxField = "base_PsfFlux_instFlux"
-config.measurePsf.starSelector["objectSize"].widthMin = 0.9
-config.measurePsf.starSelector["objectSize"].fluxMin = 4000
-
-# Set to match defaults currently used in HSC production runs (e.g. S15B)
-config.catalogCalculation.plugins['base_ClassificationExtendedness'].fluxRatio = 0.95
```

Comment:
Another modification; this now defaults to:
```
# critical ratio of model to psf flux
config.catalogCalculation.plugins['base_ClassificationExtendedness'].fluxRatio=0.985
```
Why has this been modified here?

Reply:
This value is the new task default.

Reply:
If 0.95 is the new task default, that's :+1: .

Reply:
If you want to compare this to main, I'd just build lsstsw on this branch, instead of trying to manage all the packages by yourself.

It would be easier to look at the "decam-merian" files I attached to the ticket on Jira; it has tow configs (this and main), including a diff between this branch and main. The value you're quoting above is still overridden in a few places (where we're doing cmodel), and has some other places where the value is irrelevant because the plugin isn't being run.

Reply:
I haven't had the time to build lsstsw myself, so I'll have to take your word for it that this value either a) doesn't change, or b) isn't used here (I'm still not clear on which you're advocating here).